src/Altinn.Profile/Models/OrgNotificationAddressesResponse.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+
+namespace Altinn.Profile.Models;
+
+/// <summary>
+/// A list representation of <see cref="OrganizationContactPoints"/>
+/// </summary>
+public class OrgNotificationAddressesResponse
+{
+    /// <summary>
+    /// A list containing contact points for organizations
+    /// </summary>
+    public List<OrganizationContactPoints> ContactPointsList { get; set; } = [];
+
+    /// <summary>
+    /// Class describing the contact points for an organization as notifications services uses
+    /// </summary>
+    public class OrganizationContactPoints
```

Comment:
These two are still using the term "ContactPoints". Should we update them to use NotificationAddresses?